physionet-django/physionet/views.py

```diff
@@ -1,11 +1,13 @@
 from collections import OrderedDict
 from os import path
 from re import fullmatch
+import pdb
```

Comment:
No, but 18 other files have them and it would be nice to progress so all of them have it for debug purposes

Reply:
Just because it's in our code already, doesn't mean it should be there! If you're using Python 3.7, `breakpoint()` should do what you need and doesn't require an import.

Reply:
Isn't the `venv` 3.6? I can remove it if it shouldn't be there

Reply:
It's not a big deal, but I guess my question is why we would import pdb when it isn't not used? If someone is debugging then they can `import pdb; pdb.set_trace()` wherever they want the breakpoint.

Reply:
I was just unsure of the consistency we want in the code since some files have it and some don't